### PR TITLE
uwc:cout removed from all containers

### PR DESCRIPTION
**Pull request by anujsoftdel**
-Removed cout statements and added corresponding log messages

Signed-off-by: anuj.kumar@softdel.com <anuj.kumar@softdel.com>

### CODE REVIEW

File: modbus-master/Modbus-App/src/Main.cpp [hunk 1]
```diff
@@ -94,9 +94,6 @@ void populatePollingRefData()
 		}
 		try
 		{
-			std::cout << iCount++ << ". Point to poll: " << a.getID() << ", RT: "
-					<< a.getDataPoint().getPollingConfig().m_bIsRealTime << ", Freq: "
-					<< a.getDataPoint().getPollingConfig().m_uiPollFreq << std::endl;
```
Comment (arunkk-intel) [inline modbus-master/Modbus-App/src/Main.cpp:99]: Dont we need these logs as Info logs still?

Comment (vanidesai) [reply modbus-master/Modbus-App/src/Main.cpp:99]: yes its required. Put them into logger with -Verbose settings.

**Review by vanidesai — APPROVED** [review]

File: modbus-master/Modbus-App/src/Main.cpp [hunk 2]
```diff
@@ -534,27 +511,24 @@ int main(int argc, char* argv[])
...
 		cv.wait(lck,exitMainThread);
 
 		DO_LOG_INFO("Condition variable is set for application exit.");
 
-		DO_LOG_INFO("Exiting the application gracefully.");
-		std::cout << "************************************************************************************************" <<std::endl;
-		std::cout << "********************** Exiting modbus container ***********" <<std::endl;
-		std::cout << "************************************************************************************************" <<std::endl;
+		DO_LOG_WARN("Exiting the application gracefully.");
+		DO_LOG_WARN("Exiting modbus container");
```
Comment (vkb1) [inline modbus-master/Modbus-App/src/Main.cpp:525]: I see only line 524 is sufficient, no need to talk about container here i.e., 525. You can add modules application in line 524 if required

**Review by vkb1 — APPROVED** [review]

File: modbus-master/Modbus-App/src/Main.cpp (hunk 1 above)
Comment (dspshivaji) [reply modbus-master/Modbus-App/src/Main.cpp:99]: Yes it is required. These logs are already present in DO_LOG_INFO which is captured at line number 106 – 113 as below 
DO_LOG_INFO("Polling is set for " +
a.getDataPoint().getID() +
", FunctionCode " +
std::to_string((unsigned)uiFuncCode) +
", frequency " +
std::to_string(a.getDataPoint().getPollingConfig().m_uiPollFreq) +
", RT " +
std::to_string(a.getDataPoint().getPollingConfig().m_bIsRealTime));

File: modbus-master/Modbus-App/src/Main.cpp (hunk 2 above)
Comment (nagdeepgk) [reply modbus-master/Modbus-App/src/Main.cpp:525]: @https://github.com/dspshivaji: @anujsoftdel : Is it addressed ?Can you plz address it ?

**Comment by nagdeepgk** [comment]
@anujsoftdel and @dspshivaji : Can you please confirm if for all the cout messages that you have removed, you have replaced it with corresponding logger messages because, we don't want to lose any of the prints which would be needed for debug .

**Pull request merged**